core/src/main/java/org/apache/carbondata/core/datastore/filesystem/AbstractDFSCarbonFile.java

```diff
@@ -455,7 +455,7 @@ public boolean createNewLockFile() throws IOException {
     try {
       listStatus = fileSystem.listStatus(path);
     } catch (IOException e) {
-      LOGGER.error("Exception occured: " + e.getMessage(), e);
```

Comment:
revert this change if not rerquired

Reply:
@akashrn5  Every time we create a new table/index/mv, we get this error log in the list files. This is the normal flow. Instead of error, warn is better to avoid confusion. I think, this looks ok. May be @maheshrajus can show this change details in the PR description.

Reply:
@akashrn5 @VenuReddy2103  i will add details in PR description